lexers/embedded/robot.xml

```diff
@@ -0,0 +1,81 @@
+<lexer>
+  <config>
+    <name>Robot Framework</name>
+    <alias>robot</alias>
+    <filename>*.robot</filename>
+    <mime_type>text/x-robotframework</mime_type>
+  </config>
+  <rules>
+    <state name="root">
+      <rule>
+        <include state="whitespace"/>
+      </rule>
+      <rule pattern="^\*{3}\s*Settings\s*\*{3}">
+        <token type="Keyword"/>
+      </rule>
+      <rule pattern="^\*{3}\s*Test Cases\s*\*{3}">
+        <token type="Keyword"/>
+      </rule>
+      <rule pattern="^\*{3}\s*Variables\s*\*{3}">
+        <token type="Keyword"/>
+      </rule>
+      <rule pattern="^\*{3}\s*Keywords\s*\*{3}">
+        <token type="Keyword"/>
+      </rule>
+      <rule pattern="\[Documentation\]|\[Tags\]|\[Setup\]|\[Teardown\]|\[Template\]|\[Timeout\]">
+        <token type="NameTag"/>
+      </rule>
+      <rule pattern="#.*$">
+        <token type="Comment"/>
+      </rule>
+      <rule pattern="\$\{[^\}]+\}|\@{[^\}]+\}|\&\{[^\}]+\}">
```

Comment:
I noticed this ampersand and the other one below were causing the lexer not to be loaded. It seems to work as `<rule pattern="${[^}]+}|@{[^}]+}|&amp;{[^}]+}">`

Reply:
You're right, I fixed it.